content/docs/services/elasticsearch17.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: Elasticsearch 1.7
 name: "elasticsearch17"
-description: "[Elasticsearch](https://www.elastic.co/) version 1.7"
+description: "[Elasticsearch](https://www.elastic.co/products/elasticsearch) version 1.7 a distributed, RESTful search and analytics engine"
```

Comment:
Right, the links will be stripped out for the actual service broker metadata. The links here will only be displayed on the docs site. I think we should keep them since they are useful in an HTML context.